src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-scroll';
+import logo from '../img/logo.png';
+import './styles/NavigationBar.css';
+import navigationLinks from '../data/listOfNavigation.json';
+
+const NavigationBar = () => {
+  const barLinks = [];
+  navigationLinks.forEach((item, index) => {
+    if (index) {
+      barLinks.push(<span className="line left-line"></span>);
+    }
```

Comment:
Ya se me ocurrió como ponerlo con el operador ternario xD
```suggestion
   index ? barLinks.push(<span className="line left-line"></span>) : null;
```